ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -116,6 +116,19 @@ def test_numpy_ndarray_property_T(
     )
 
 
+@given(
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes(kind="numeric", prune_function=False),
+        ret_shape=True,
+    ),
+)
+def test_numpy_ndarray_property_flat(dtypes_and_x):
```

Comment:
The way you're manually handling the tests run is incorrect we should be computing the returns of both the numpy framework and ivy frontends, and then we can compare the results and check if they're the same.